react/features/base/util/openURLInBrowser.web.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+/**
+ * Opens URL in the browser.
+ *
+ * @param {string} url - The URL to be opened.
+ * @returns {void}
+ */
+export function openURLInBrowser(url: string) {
+    window.open(url);
```

Comment:
What happened to Lenny's comment about noopener. I think we should set it. See https://developer.mozilla.org/en-US/docs/Web/API/Window/open

Reply:
fixed